src/neighbors.cc

```diff
@@ -594,16 +545,14 @@ neighborsUdpPing(HttpRequest * request,
 {
     const char *url = entry->url();
     MemObject *mem = entry->mem_obj;
-    CachePeer *p = nullptr;
-    int i;
     int reqnum = 0;
     int flags;
     int peers_pinged = 0;
     int parent_timeout = 0, parent_exprep = 0;
     int sibling_timeout = 0, sibling_exprep = 0;
     int mcast_timeout = 0, mcast_exprep = 0;
 
-    if (Config.peers == nullptr)
+    if (!CurrentCachePeers().size())
```

Comment:
Let's undo this change if it is not necessary:

```suggestion
    if (Config.peers == nullptr)
```